packages/data-format/src/format/rnf_invoice/types.ts

```diff
@@ -1,6 +1,6 @@
 export interface Invoice {
   meta: {
-    format: 'rnf_invoice';
+    format: 'rnf_invoice' | 'rnf_generic';
```

Comment:
I would like more explanation about this, please. It looks strange for an interface `Invoice` to support both formats.

Reply:
The rnf_generic type is identical to the rnf_invoice type so we can reuse the same interface vs creating a new one.

Reply:
Completely identical? Why do we have a new format if it's the same?

Reply:
It's much easier to handle things in the frontend, everything in API-sync like emails + same with portal. Long term it's much cleaner this way.

Reply:
Easier for you but the library should be clear for everyone using / implementing the protocol.

What about extending a common interface (like `InvoiceData` ?) to `Invoice` and `Generic` (or, to be more precise `InvoiceBasedGeneric`) so it's clear that the only difference is their name and format ? The front-end could then rely on the common interface or on `Invoice | Generic` and it would not be more difficult.

Reply:
can't we have `rnf_generic` without a change to the protocol? 

This is an experiment, not something we want very rigorously in the protocol I feel. I even think it's fine skipping the validation for this one if it helps use move forwared

Reply:
> can't we have `rnf_generic` without a change to the protocol?
> 
> This is an experiment, not something we want very rigorously in the protocol I feel. I even think it's fine skipping the validation for this one if it helps use move forwared

We can bypass the validation if you think that's best?